addons/wowl/static/src/registries.ts

```diff
@@ -134,3 +165,40 @@ userMenuRegistry
   .add("profile", preferencesItem)
   .add("odoo_account", odooAccountItem)
   .add("log_out", logOutItem);
+
+// -----------------------------------------------------------------------------
+// Default DebugManagerMenu items
+// -----------------------------------------------------------------------------
+
+export const debugManagerMenuRegistry: Registry<DebugManagerMenuItemFactory> = new Registry();
+
+debugManagerMenuRegistry
+  .add("js_tests", runJSTestsItem)
+  .add("js_tests_mobile", runJSTestsMobileItem)
+  .add("click_test", runClickTestItem)
+  .add("open_view", openViewItem)
+
+  .add("action_separator", actionSeparator)
+  .add("edit_action", editAction)
+  .add("view_fields", viewFields)
+  .add("manage_filters", manageFilters)
+  .add("technical_translation", technicalTranslation)
+
+  .add("access_separator", accessSeparator)
+  .add("view_access_right", viewAccessRights)
+  .add("view_record_rules", viewRecordRules)
+
+  .add("view_separator", viewSeparator)
+  .add("set_defaults", setDefaults)
+  .add("view_metadata", viewMetadata)
+  .add("manage_attachments", manageAttachments)
+  .add("fields_view_get", fieldsViewGet)
+  .add("edit_view", editView)
+  .add("edit_control_panel_view", editControlPanelView)
+
+  .add("global_separator", globalSeparator)
+  .add("activate_assets", activateAssetsDebugging)
+  .add("activate_tests_assets", activateTestsAssetsDebugging)
+  .add("regenerate_assets", regenerateAssets)
+  .add("become_superuser", becomeSuperuser)
+  .add("leave_debug_mode", leaveDebugMode);
```

Comment:
Not sure I want to see all this in this file. An alternative would be to populate an array of items in the debugmanager file, export it, import it here and iterate on it to add each item it contains to the registry.

That being said, I'm not sure we'll keep this file, defining and populating all registries.